doc/spec.md

```diff
@@ -2596,17 +2600,58 @@ Each property assignment in an object literal is processed as follows:
+### Object literal accessors { #object-literal-accessors }
+
+A get accessor declaration is processed in the same manner as an ordinary function declaration (section [6.1](#6.1)) with no parameters.
+A set accessor declaration is processed in the same manner as an ordinary function declaration with a single parameter and a Void return type.
+When both a get and set accessor is declared for a property:
 
 * If both accessors include type annotations, the specified types must be identical.
 * If only one accessor includes a type annotation, the other behaves as if it had the same type annotation.
 * If neither accessor includes a type annotation, the inferred return type of the get accessor becomes the parameter type of the set accessor.
 
 If a get accessor is declared for a property, the return type of the get accessor becomes the type of the property. If only a set accessor is declared for a property, the parameter type (which may be type Any if no type annotation is present) of the set accessor becomes the type of the property.
 
-When an object literal is contextually typed by a type that includes a string index signature, the resulting type of the object literal includes a string index signature with the union type of the types of the properties declared in the object literal, or the Undefined type if the object literal is empty. Likewise, when an object literal is contextually typed by a type that includes a numeric index signature, the resulting type of the object literal includes a numeric index signature with the union type of the types of the numerically named properties (section [3.9.4](#3.9.4)) declared in the object literal, or the Undefined type if the object literal declares no numerically named properties.
+### Object literal spread assignment { #object-literal-spread-assignment }
+
+A spread assignment inserts properties into the object literal as if they all were written at the point of the spread assignment.
+Unlike normal property assignments, however, duplicates are allowed.
+In the case of duplicates, the last assignment is the one that is that in the resulting object type.
+
+The only allowed types of expressions allowed in a spread assignment are
+  * The any type.
+  * The number, boolean, string or non-primitive object types.
+  * Mapped types.
+  * Unions or intersections of the above types.
+Spreading other types results in an error.
+
+When a spread assignment in encountered in an object literal, the resulting type is defined as a combination of the type of the spread assignment and the object type built from the assignments preceding the spread assignment.
+For a preceding object type *O*, spreading:
+
+* The any type results in the any type.
+* The never type results in *O*.
+* A union type *U* results in the union of `{ O, ...u }` for all *u* in *U*.
```

Comment:
Might be clearer to use `*T*` instead of `*u*`.

Reply:
Can we pleeeeeeease use math markup to say ${ O, ...u } \forall u \in U$ ?

In the meantime, you are right.